src/components/Table/style.css

```diff
@@ -0,0 +1,3 @@
+.table-column-warning {
```

Comment:
in questo caso la tua classe dovrebbe essere creata in questo modo 
.tableColumnWarning per essere importata con `import {tableColumnWarning} from 'table.mocule.css'`

Reply:
però non capisco. usiamo la row class name ma la classe verte sulla column?

Reply:
Errore mio, la classe verte sulla riga, è il valore della colonna che la fa applicare o meno, rinomino la classe